week1/pages/add.js

```diff
@@ -0,0 +1,16 @@
+
+import HTTP from 'http'
+import STATUS  from './status.js'
+ 
+ 
+
+module.exports = {
+    "body" : function contentHomePage(Response ){
+        STATUS.varia = STATUS.varia + 1 ;
+        Response.write('<h1>add one</h1>')
+        console.log('this is the value of varia '+ STATUS.varia)
+        Response.write(STATUS.varia.toString())
+        Response.end();
```

Comment:
You are calling `response.end()` in every page, but also in the `index.js`. Maybe you could remove it from the page functions. It should still work, right?